statick_tool/plugins/tool/cccc_tool_plugin.py

```diff
@@ -77,7 +77,7 @@ def scan(  # pylint: disable=too-many-branches
                 print("Couldn't find cccc executable! ({})".format(ex))
                 return None
 
-            if self.plugin_context.args.show_tool_output:
+            if self.plugin_context and self.plugin_context.args.show_tool_output:
```

Comment:
this tool checks for self.plugin_context is None at the top of this function. is this change still necessary?

Reply:
Statick can still crash if `self.plugin_context` is empty. I've seen that happen a few times in the past.